public/javascripts/script.js

```diff
@@ -19,13 +39,34 @@ addFood(steak[0], '#steak', () => {
 // Iteration 2 using `.then()`
 addFood(mashPotatoes[0], '#mashPotatoes').then(() => {
   // ... your code here
-  addFood(mashPotatoes[1], '#mashPotatoes')
+  addFood(mashPotatoes[1], '#mashPotatoes').then(()=>{
```

Comment:
In order to make the code looks cleaner you can use this structure: https://prnt.sc/u4u573

Reply:
Same in the following lines